src/transformers/models/decision_transformer/modeling_decision_transformer.py

```diff
@@ -746,7 +747,8 @@ class DecisionTransformerPreTrainedModel(PreTrainedModel):
     base_model_prefix = "decision_transformer"
     main_input_name = "states"
     supports_gradient_checkpointing = False
-    _keys_to_ignore_on_load_missing = [r"position_ids"]
+    _keys_to_ignore_on_load_missing = [r"position_ids", r"h\.\d+\.attn\.masked_bias", r"h\.\d+\.attn\.bias"]
```

Comment:
Should `r"h\.\d+\.attn\.masked_bias", r"h\.\d+\.attn\.bias"` be in ` _keys_to_ignore_on_load_missing`?